packages/ui-extensions/src/docs/shared/components/Banner.ts

```diff
@@ -0,0 +1,11 @@
+import {SharedReferenceEntityTemplateSchema} from '../docs-type';
+
+const data: SharedReferenceEntityTemplateSchema = {
+  name: 'Banner',
+  description:
```

Comment:
to avoid the `\n\n` we can write this descriptions as markdown so 
```suggestion
  description:`
  Use s-banner... .
  You can use...
 `
```

we have that example on clipboard item

Reply:
yeah, I'll use markdown when we rewrite them.